src/tests/dbus-tests/udiskstestcase.py

```diff
@@ -80,9 +88,6 @@ def get_version():
         pretty_name = str(sys_info.Get("org.freedesktop.hostname1", "OperatingSystemPrettyName", dbus_interface=dbus.PROPERTIES_IFACE))
         distro, version = get_version_from_pretty_name(pretty_name)
 
-    # we want just the major version, so remove all decimal places (if any)
-    version = str(int(float(version)))
```

Comment:
Isn't this still important for the other methods?

Reply:
Fixed. I've just copy-pasted the functions from libblockdev without checking. Only the cpe name contains minor versions (and only on RHEL).